lib/src/viewer.dart

```diff
@@ -225,7 +225,7 @@ class _PDFViewerState extends State<PDFViewer> {
             )
           : null,
       floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
-      bottomNavigationBar: (widget.showNavigation || widget.document.count > 1)
+      bottomNavigationBar: widget.showNavigation
```

Comment:
Instead of removing the 2nd part of the condition, we can just change OR to AND.